docs/source/challenge-doc.rst

```diff
@@ -32,7 +32,7 @@ Objective
 
 
 Move the object between two locations with a handover between a hand and a prosthesis. The task parameters will be randomized to provide a comprehensive 
-test to the model performance. The randomization will include but not limited to: object type, object weight and even friction during each environmental reset. 
+test to the controller model performance. The randomization will include but not limited to: object type, object weight and even friction during each environmental reset. 
```

Comment:
controller model performance => controller's performance / Policy's performance

Reply:
updated in commit [commit](https://github.com/MyoHub/myosuite/commit/e31145ea1a059add2cb32370ebb6951a8b3071a7)